src/Pack/Config/Types.idr

```diff
@@ -382,6 +401,7 @@ update ci cm = MkConfig {
   , useKatla        = fromMaybe ci.useKatla cm.useKatla
   , withIpkg        = fromMaybe ci.withIpkg cm.withIpkg
   , rlwrap          = fromMaybe ci.rlwrap cm.rlwrap
+  , extraArgs       = ci.extraArgs <+> fromMaybe NoExtraArgs cm.extraArgs
```

Comment:
BTW this might be a questionable choice, so please let me know if any other strategy of merging the two configs would be better here!

Reply:
I think we should join them, but we need to assure that values from the localmost configs would be the rightmost

Reply:
Hmm, just to confirm — what'd be the localmost config in this case? The `Maybe`-indexed one?

Reply:
Cool, then this already should be the case, I guess.